src/autopas/containers/cellPairTraversals/SlicedBasedTraversal.h

```diff
@@ -244,18 +255,18 @@ void SlicedBasedTraversal<ParticleCell, PairwiseFunctor, dataLayout, useNewton3>
       // at the end of the first layers release the lock
       if (slice > 0 && dimSlice < myStartArray[_dimsPerLength[0]] + _overlapLongestAxis) {
         const unsigned long index = ((slice - 1) * _overlapLongestAxis) + (dimSlice - myStartArray[_dimsPerLength[0]]);
-        locks[index].unlock();
-        // if lastLayer is reached within overlap area, unlock all following locks
+        _locks[index].unlock();
+        // if lastLayer is reached within overlap area, unlock all following _locks
         if (dimSlice == lastLayer - 1) {
           for (unsigned long i = dimSlice + 1; i < myStartArray[_dimsPerLength[0]] + _overlapLongestAxis; ++i) {
             const unsigned long index = ((slice - 1) * _overlapLongestAxis) + (i - myStartArray[_dimsPerLength[0]]);
```

Comment:
yes again not your code but since you already work on those files... ;)
This variable shadows another one.